ldap/georchestra.ldif

```diff
@@ -187,4 +187,28 @@ ou: 23
 description: This group grants basic, authenticated access to the whole SDI
 member: uid=testuser,ou=users,dc=georchestra,dc=org
 
+# orgs, georchestra.org
+dn: ou=orgs,dc=georchestra,dc=org
+objectClass: top
+objectClass: organizationalUnit
+ou: orgs
+
+dn: o=onf,ou=orgs,dc=georchestra,dc=org
+objectClass: top
+objectClass: organization
+o: onf
+description: association
+postalAddress: 127 rue georchestra, 73590 Chamblille
+
+dn: cn=onf,ou=orgs,dc=georchestra,dc=org
+objectClass: top
+objectClass: groupOfMembers
+cn: onf
+businessCategory: inscrit
```

Comment:
I'd prefer `businessCategory: REGISTERED` (camelcase to indicate that this is a value from a static list)